### PR TITLE
Docs-10865 document se linux requirement

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/install-aws.adoc [hunk 1]
```diff
@@ -141,17 +141,20 @@ Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
 |Variable | Description | Example
 | `block_aws_metadata_svc` | A boolean variable that blocks the AWS metadata service on the host operating system. The default value is `false`. | `true`
 | `controllers` | The number of controller machines to provision. The default value is `3`. | `5`
-| `disable_selinux` | A boolean variable that enables/disables SELinux on the host operating system. The default value is `true`. | `true`
+| `disable_selinux` | A boolean variable that disables/enables SELinux on the host operating system. The default value is `true` to disable SELinux.
+{sp} +
+{sp} +
+If the host operating system is RHEL 8, do not set to `false` or installation will fail. | `true`
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/install-aws.adoc:147]: @kmendiratta-crm I just ended up adding this to the table cell.

**Comment by kmendiratta-crm** [comment]
@IsaacEldridge https://docs.mulesoft.com/runtime-fabric/1.9/install-azure#deploy-using-the-arm-template & https://docs.mulesoft.com/runtime-fabric/1.9/install-manual#variables-to-define will need updating too.

**Review by vli63127 — DISMISSED** [review]

**Review by kmendiratta-crm — APPROVED** [review]

File: modules/ROOT/pages/install-azure.adoc [hunk 2]
```diff
@@ -121,25 +121,28 @@ create a separate Resource group.
 | `Availability Set Fault Domains` | specifies the number of fault domains supported in the Azure region selected. | `3`
 | `Pod Network CIDR` | specifies a custom CIDR block to use for the pod network. | `10.244.0.0/16`
 | `Service CIDR` | specifies a custom CIDR block to use for the service network. | `10.100.0.0/16`
-
+| `disable_selinux` | A boolean variable that disables/enables SELinux on the host operating system. The default value is `true` to disable SELinux.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/install-azure.adoc:124]: docs review lines 124-127

File: modules/ROOT/pages/install-manual.adoc [hunk 3]
```diff
@@ -123,16 +123,19 @@ The following variables can be defined based upon your organization's requiremen
 | `RTF_SERVICE_GID` | An integer representing the group ID used when running each Runtime Fabric service. Overrides the default behavior of creating a group named "planet". | `900`
 | `POD_NETWORK_CIDR` | Support for a custom pod CIDR block. | `10.244.0.0/16`
 | `SERVICE_CIDR` | Support for a custom service CIDR block. | `10.100.0.0/16`
-
+| `disable_selinux` | A boolean variable that disables/enables SELinux on the host operating system. The default value is `true` to disable SELinux.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/install-manual.adoc:126]: docs review lines 126-129

File: modules/ROOT/pages/runtime-fabric-limitations.adoc [hunk 4]
```diff
@@ -35,6 +44,15 @@ The internal load balancer runs on the controller nodes of Runtime Fabric on VMs
 type of inbound traffic. You can allocate only half of the available CPU cores on each VM to the internal load balancer.
 Refer to xref:deploy-resource-allocation#internal-load-balancer[Resource Allocation and Performance on Anypoint Runtime Fabric] for additional information.
 
+== Operating System Limitations
+
+The following table lists known limitations for host operating systems in Runtime Fabric:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-fabric-limitations.adoc:49]: docs review lines 49-53

**Review by dukesphere — APPROVED** [review]

**Pull request merged**